packages/account/src/Components/file-uploader-container/file-uploader-container.tsx

```diff
@@ -63,77 +33,25 @@ const FileUploaderContainer = ({
-                        <Icon icon='IcImage' className='account-poa__upload-icon' size={24} />
-                        <div className='account-poa__upload-item'>
-                            <Localize i18n_default_text='JPEG JPG PNG PDF GIF' />
-                        </div>
-                    </li>
-                    <li className='account-poa__upload-box'>
-                        {is_appstore ? (
-                            <Icon icon='IcPoaFileTime' className='account-poa__upload-icon' size={24} />
-                        ) : (
-                            <Icon icon='IcClock' className='account-poa__upload-icon' size={24} />
-                        )}
-                        <div className='account-poa__upload-item'>
-                            <Localize i18n_default_text='1 - 6 months old' />
-                        </div>
-                    </li>
-                    <li className='account-poa__upload-box'>
-                        {is_appstore ? (
-                            <Icon icon='IcPoaFileClear' className='account-poa__upload-icon' size={24} />
-                        ) : (
-                            <Icon icon='IcEye' className='account-poa__upload-icon' size={24} />
-                        )}
-                        <div className='account-poa__upload-item'>
-                            <Localize i18n_default_text='A clear colour photo or scanned image' />
-                        </div>
-                    </li>
-                    <li className='account-poa__upload-box'>
-                        {is_appstore ? (
-                            <Icon icon='IcPoaFileWithAddress' className='account-poa__upload-icon' size={24} />
-                        ) : (
-                            <Icon icon='IcUser' className='account-poa__upload-icon' size={24} />
-                        )}
-                        <div className='account-poa__upload-item'>
-                            <Localize i18n_default_text='Issued under your name with your current address' />
-                        </div>
-                    </li>
-                </ul>
-            )}
-            <div
-                className={classNames('account-poa__upload-file', {
-                    'account-poa__upload-file--dashboard': isDesktop() && is_appstore,
-                })}
-            >
+        <div className='file-uploader__container' data-testid='dt_file_uploader_container'>
+            {files_description}
+            <Text size={isMobile() ? 'xxs' : 'xs'} as='div' className='file-uploader__file-title' weight='bold'>
+                <Localize i18n_default_text='Upload file' />
+            </Text>
+            <div className='file-uploader__file-dropzone-wrapper'>
                 <FileUploader getSocket={getSocketFunc} ref={ref} onFileDrop={onFileDrop} settings={settings} />
             </div>
+            <div className='file-uploader__file-supported-formats'>
+                <Text size={isMobile() ? 'xxxs' : 'xxs'}>
```

Comment:
@yauheni-deriv i would map these Texts but there are only 2 of them, so it's up to you :)